fda-model/structures/Inventory.cs

```diff
@@ -223,7 +223,7 @@ internal void GetUniqueDamageCatagories()
         _damageCategories = damageCatagories;
     }
 
-    private Inventory GetInventoryTrimmmedToPolygon(int impactAreaFID)
+    public Inventory GetInventoryTrimmmedToPolygon(int impactAreaFID)
```

Comment:
are we confident that the impact area ID that we track in the model is the same as the impactAreaFID?

Reply:
I remember getting myself confused on this, and I don't remember where we left off.